apps/condo/domains/common/constants/style.js

```diff
@@ -44,6 +44,7 @@ const successBG = '#EBFAEF'
 const warningBg = '#FFECEB'
 const warningText = '#FF3B30'
 const infoIconColor = '#39CE66'
+const warningAlert = '#F086331A'
```

Comment:
it's strange...

Reply:
+1

Reply:
The last 2 symbols are alpha (transparency). Looks like this value was copied from Figma.

Reply:
@AleX83Xpert Yes, this value was copied from Figma. But we came to the conclusion that the format `rgba(r, g, b, a)` looks comfortable